lib/rubill/base.rb

```diff
@@ -49,7 +49,13 @@ def self.delete(id)
     end
 
     def self.where(filters=[])
-      raise ArgumentError unless filters.all? { |f| f.is_a?(Query::Filter) }
+      raise ArgumentError unless filters.is_a?(Hash) || filters.all? { |f| f.is_a?(Query::Filter) }
```

Comment:
I should've been more specific: this will raise a `NoMethodError` when filters gets sent `#all?`. It's probably not that big of a deal as most Ruby code has this problem anyways, so unless you feel compelled to change it I'm okay with how it is.